manifests/example/kube-applier-patch.yaml

```
@@ -11,6 +10,8 @@ spec:
         env:
         - name: REPO_PATH
           value: "/src/manifests/example-env/"
+        - name: REPO_PATH_FILTERS
+          value: "my-namespace-1,my-namespace-2"
```

```suggestion
          value: "my-namespace-1,my-namespace-2,team-namespace-*"
```
to show support for wildcard